gradle/publish.gradle

```diff
@@ -1,9 +1,10 @@
-apply from: "${rootDir}/gradle/publish-base.gradle"
+apply from: "${rootDir}/gradle/publish-jar.gradle"
 
-publishing {
-    publications {
-        nebula(MavenPublication) {
-            from components.java
-        }
+jar {
+    manifest {
+        attributes(
+                "Implementation-Title"   : project.name,
+                "Implementation-Version" : project.version,
+                "Implementation-Vendor"  : "Palantir Technologies Inc.")
```

Comment:
I don't think Implementation-Title or Vendor are used for anything.  Can we delete and stick to just the publish-jar.gradle file?